src/libs/OptionsListUtils.js

```diff
@@ -82,9 +82,16 @@ function createOption(personalDetailList, report, draftComments, activeReportID)
         && (report.reportID !== activeReportID)
         && lodashGet(draftComments, `${ONYXKEYS.COLLECTION.REPORT_DRAFT_COMMENT}${report.reportID}`, '').length > 0;
 
+    const lastActorDetails = report ? personalDetailList.find(({login}) => login === report.lastActorEmail) : null;
     return {
         text: report ? report.reportName : personalDetail.displayName,
-        alternateText: (report && hasMultipleParticipants) ? report.reportName : personalDetail.login,
+        lastMessageText: report
+            ? (hasMultipleParticipants && lastActorDetails
+                ? `${lastActorDetails.displayName}: `
+                : '')
+              + report.lastMessageText
+            : '',
+        lastActorEmail: report ? report.lastActorEmail : null,
```

Comment:
I think there should be no need to supply this as an option parameter. If we are mainly doing this so that we can make this change occur in the ChatSwitcherView there's no need as it will be removed very soon.

Reply:
This function already contains the report check on every field so I have followed the pattern and put a check for the report but as the last message text is a string field so I gave `' '` But I am changing it to `null`.

Reply:
Sorry for the confusion, I am trying to say that `lastActorEmail` is not needed to be passed in the object returned from `createOption()`.

Reply:
ok

Reply:
Ok